packages/odyssey-react-mui/src/labs/DataComponents/StackCard.tsx

```diff
@@ -0,0 +1,194 @@
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import {
+  MouseEventHandler,
+  ReactElement,
+  memo,
+  useMemo,
+  ReactNode,
+} from "react";
+import {
+  Card as MuiCard,
+  CardActions as MuiCardActions,
+  CardActionArea as MuiCardActionArea,
+} from "@mui/material";
+import styled from "@emotion/styled";
+
+import { Button } from "../../Button";
+import { ButtonContext } from "../../ButtonContext";
+import { MoreIcon } from "../../icons.generated";
+import { MenuButton, MenuButtonProps } from "../../MenuButton";
+import {
+  DesignTokens,
+  useOdysseyDesignTokens,
+} from "../../OdysseyDesignTokensContext";
+import { Heading5, Paragraph, Support } from "../../Typography";
+import { Box } from "../../Box";
+
+export const CARD_IMAGE_HEIGHT = "64px";
+
+export type StackCardProps = {
+  description?: string;
+  image?: ReactElement;
+  overline?: string;
+  title?: string;
+  children?: ReactNode;
+} & (
+  | {
+      onClick: MouseEventHandler;
+      button?: never;
+      menuButtonChildren?: never;
+      Accessory?: never;
+    }
+  | {
+      onClick?: never;
+      button?: ReactElement<typeof Button>;
+      menuButtonChildren?: MenuButtonProps["children"];
+      Accessory?: ReactNode;
+    }
+);
+
+const ImageContainer = styled("div", {
+  shouldForwardProp: (prop) =>
+    prop !== "odysseyDesignTokens" && prop !== "hasMenuButtonChildren",
+})<{
+  odysseyDesignTokens: DesignTokens;
+  hasMenuButtonChildren: boolean;
+}>(({ odysseyDesignTokens, hasMenuButtonChildren }) => ({
+  display: "flex",
+  alignItems: "flex-start",
+  maxHeight: `${CARD_IMAGE_HEIGHT}`,
+  marginBlockEnd: odysseyDesignTokens.Spacing5,
+  paddingRight: hasMenuButtonChildren ? odysseyDesignTokens.Spacing5 : 0,
+}));
+
+const MenuButtonContainer = styled("div", {
+  shouldForwardProp: (prop) => prop !== "odysseyDesignTokens",
+})<{ odysseyDesignTokens: DesignTokens }>(({ odysseyDesignTokens }) => ({
+  position: "absolute",
+  right: odysseyDesignTokens.Spacing3,
+  top: odysseyDesignTokens.Spacing3,
+}));
+
+const buttonProviderValue = { isFullWidth: true };
+
+const StackCard = ({
+  button,
+  description,
+  image,
+  menuButtonChildren,
+  onClick,
+  overline,
+  title,
+  children,
+  Accessory,
+}: StackCardProps) => {
+  const odysseyDesignTokens = useOdysseyDesignTokens();
+
+  const cardContent = useMemo(
+    () => (
+      <Box
+        sx={{
+          display: "flex",
+        }}
+      >
+        {Accessory && (
+          <Box
+            sx={{
+              marginInlineEnd: 3,
+              marginInlineStart: -2,
```

Comment:
negative margin feels weird/kinda gross here. Is there a better way to fix whatever issue we were fixing here?